jobbers/templates/femfat_lsf.j2

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#BSUB -J {{ jobname }}
+##BSUB -n {{ node_cores }}
+#BSUB -n 1
+#BSUB -R "select[n_femfat]"
+#BSUB -q debug
+##BSUB -x
+
+{{ start }}
+
+export PATH=/share/apps/femfat/Linux/femfat{{ version }}/bin:$PATH
+echo $PATH > path.txt
+
+# unset SLURM_GTIDS (Seems abaqus crashes when this is set)
+unset SLURM_GTIDS
+
+export MAGNAECS_LICENSE_PATH="6300@licserv0003.scania.com:6300@licserv0002.scania.com:6300@licserv0001.scania.com"
```

Comment:
This is a reference to scania specific server infrastructure. Can this be moved into a separate facility?